contrib/demo/ingress.result

```diff
@@ -0,0 +1,51 @@
+ingress-demo $ [1;37mkubectl get ingresses --all-namespaces[0m
```

Comment:
is this used anywhere or just for documentation?

Reply:
the demo output is logged and then "compared" with this file to decide if it has been successful or not.